service/src/main/java/bio/terra/landingzone/service/iam/LandingZoneSamClient.java

```diff
@@ -0,0 +1,51 @@
+package bio.terra.landingzone.service.iam;
+
+import bio.terra.common.tracing.OkHttpClientTracingInterceptor;
+import bio.terra.landingzone.library.configuration.LandingZoneSamConfiguration;
+import io.opencensus.trace.Tracing;
+import java.util.List;
+import okhttp3.OkHttpClient;
+import org.broadinstitute.dsde.workbench.client.sam.ApiClient;
+import org.broadinstitute.dsde.workbench.client.sam.api.ResourcesApi;
+import org.broadinstitute.dsde.workbench.client.sam.api.UsersApi;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LandingZoneSamClient {
+  private final LandingZoneSamConfiguration samConfig;
+  private final OkHttpClient okHttpClient;
+
+  public LandingZoneSamClient(LandingZoneSamConfiguration samConfig) {
+    this.samConfig = samConfig;
+    this.okHttpClient = new ApiClient().getHttpClient();
```

Comment:
Does it make sense to inject httpClient too?

Reply:
I don't see use of it for Sam Service test. We may add it later if needed.